From 018f920fe197ac8c6410df4f6c75c9ffa2cce6a7 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 26 Aug 2013 13:39:00 +0800
Subject: [PATCH 242/602] ENGR00276651 [MX6Q/DL SD]create empty
 fstab_nand.freescale for default build

Only touch the fstab file if it exist for FS env changing

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/arm2_6dq.mk       | 5 +++++
 imx6/arm2_6sl.mk       | 5 +++++
 imx6/evk_6sl.mk        | 5 +++++
 imx6/hdmidongle_6dq.mk | 5 +++++
 imx6/sabreauto_6q.mk   | 5 +++++
 imx6/sabresd_6dq.mk    | 5 +++++
 6 files changed, 30 insertions(+)

diff --git a/imx6/arm2_6dq.mk b/imx6/arm2_6dq.mk
index d4d03cb..e730994 100644
--- a/imx6/arm2_6dq.mk
+++ b/imx6/arm2_6dq.mk
@@ -3,8 +3,13 @@
 
 $(call inherit-product, device/fsl/imx6/imx6.mk)
 
+ifneq ($(wildcard device/fsl/arm2_6dq/fstab_nand.freescale),)
 $(shell touch device/fsl/arm2_6dq/fstab_nand.freescale)
+endif
+
+ifneq ($(wildcard device/fsl/arm2_6dq/fstab.freescale),)
 $(shell touch device/fsl/arm2_6dq/fstab.freescale)
+endif
 
 # Overrides
 PRODUCT_NAME := arm2_6dq
diff --git a/imx6/arm2_6sl.mk b/imx6/arm2_6sl.mk
index e8fbaec..73c82c9 100644
--- a/imx6/arm2_6sl.mk
+++ b/imx6/arm2_6sl.mk
@@ -3,8 +3,13 @@
 
 $(call inherit-product, device/fsl/imx6/imx6.mk)
 
+ifneq ($(wildcard device/fsl/arm2_6sl/fstab_nand.freescale),)
 $(shell touch device/fsl/arm2_6sl/fstab_nand.freescale)
+endif
+
+ifneq ($(wildcard device/fsl/arm2_6sl/fstab.freescale),)
 $(shell touch device/fsl/arm2_6sl/fstab.freescale)
+endif
 
 # Overrides
 PRODUCT_NAME := arm2_6sl
diff --git a/imx6/evk_6sl.mk b/imx6/evk_6sl.mk
index d517308..200d902 100644
--- a/imx6/evk_6sl.mk
+++ b/imx6/evk_6sl.mk
@@ -4,8 +4,13 @@
 $(call inherit-product, device/fsl/imx6/imx6.mk)
 $(call inherit-product-if-exists,vendor/google/products/gms.mk)
 
+ifneq ($(wildcard device/fsl/evk_6dq/fstab_nand.freescale),)
 $(shell touch device/fsl/evk_6sl/fstab_nand.freescale)
+endif
+
+ifneq ($(wildcard device/fsl/evk_6dq/fstab.freescale),)
 $(shell touch device/fsl/evk_6sl/fstab.freescale)
+endif
 
 # Overrides
 PRODUCT_NAME := evk_6sl
diff --git a/imx6/hdmidongle_6dq.mk b/imx6/hdmidongle_6dq.mk
index a7ddc68..b76fe3f 100644
--- a/imx6/hdmidongle_6dq.mk
+++ b/imx6/hdmidongle_6dq.mk
@@ -4,8 +4,13 @@
 $(call inherit-product, device/fsl/imx6/imx6.mk)
 $(call inherit-product-if-exists,vendor/google/products/gms.mk)
 
+ifneq ($(wildcard device/fsl/hdmidongle_6dq/fstab_nand.freescale),)
 $(shell touch device/fsl/hdmidongle_6dq/fstab_nand.freescale)
+endif
+
+ifneq ($(wildcard device/fsl/hdmidongle_6dq/fstab.freescale),)
 $(shell touch device/fsl/hdmidongle_6dq/fstab.freescale)
+endif
 
 # Overrides
 PRODUCT_NAME := hdmidongle_6dq
diff --git a/imx6/sabreauto_6q.mk b/imx6/sabreauto_6q.mk
index e836c3a..9474f81 100644
--- a/imx6/sabreauto_6q.mk
+++ b/imx6/sabreauto_6q.mk
@@ -3,8 +3,13 @@
 
 $(call inherit-product, device/fsl/imx6/imx6.mk)
 
+ifneq ($(wildcard device/fsl/sabreauto_6dq/fstab_nand.freescale),)
 $(shell touch device/fsl/sabreauto_6q/fstab_nand.freescale)
+endif
+
+ifneq ($(wildcard device/fsl/sabreauto_6dq/fstab.freescale),)
 $(shell touch device/fsl/sabreauto_6q/fstab.freescale)
+endif
 
 # Overrides
 PRODUCT_NAME := sabreauto_6q
diff --git a/imx6/sabresd_6dq.mk b/imx6/sabresd_6dq.mk
index 1629654..c43cad0 100644
--- a/imx6/sabresd_6dq.mk
+++ b/imx6/sabresd_6dq.mk
@@ -4,8 +4,13 @@
 $(call inherit-product, device/fsl/imx6/imx6.mk)
 $(call inherit-product-if-exists,vendor/google/products/gms.mk)
 
+ifneq ($(wildcard device/fsl/sabresd_6dq/fstab_nand.freescale),)
 $(shell touch device/fsl/sabresd_6dq/fstab_nand.freescale)
+endif
+
+ifneq ($(wildcard device/fsl/sabresd_6dq/fstab.freescale),)
 $(shell touch device/fsl/sabresd_6dq/fstab.freescale)
+endif
 
 # Overrides
 PRODUCT_NAME := sabresd_6dq
-- 
1.8.0

